docs/examples/api-reference/client/commit.py

```diff
@@ -36,9 +36,9 @@ class Transaction:
 
     @featureset
     class TransactionFeatures:
-        txid: int = feature(id=1)
-        amount: int = feature(id=2).extract(field=Transaction.amount, default=0)
-        amount_is_high: bool = feature(id=3)
+        txid: int = F()
+        amount: int = F(ref=Transaction.amount, default=0)
```

Comment:
I think the conclusion was to not use any kwarg but the first position arg to be the expression? So no `ref` here?

Reply:
will replace this